spec/facility_spec.rb

```diff
@@ -23,4 +37,122 @@
       expect(@facility.services).to eq(['New Drivers License', 'Renew Drivers License', 'Vehicle Registration'])
     end
   end
+
+  describe '#add a service to a facility' do
+    it 'adds Vehicle Registration to facility 1' do
+      @facility_1.add_service('Vehicle Registration')
+      expect(@facility_1.services).to eq(['Vehicle Registration'])
+    end
+  end
+
+  describe '#registered vehicles' do
+    it 'registered vehicles is an array' do
+      expect(@facility_1.registered_vehicles).to eq([])
+    end
+  end
+
+  describe '#collected fees' do
+    it 'collected fees is 0' do
```

Comment:
This test is probably unnecessary. Assert that fees are 0 before registering the vehicle in the next test!